draft-irtf-cfrg-hpke.md

```diff
@@ -710,11 +715,54 @@ byte strings for public keys.
 | 0x0002 | AES-GCM-256      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
 
-# Security Considerations
-
-The general security properties of HPKE are described in
-{{security-properties}}.  In this section, we consider a security issue that may
-arise in practice and an advanced use case.
+# Security Considerations {#sec-considerations}
+
+## Security Properties {#sec-properties}
+
+HPKE makes the following cryptographic assumptions about KDF interfaces:
+
+- Hash: Collision resistance.
+- Extract: Indifferentiable from a random oracle.
+- Expand: Behaves as a pseudorandom function wherein the first
+argument is the key.
```

Comment:
Don't you need an assumption on the AEAD too?  Presumably just IND-CCA2

Reply:
I'll add something, sure.